content/cli/commands/cluster.md

```diff
@@ -1,5 +1,266 @@
+✅ ok: Kubernetes config is loadable
+✅ ok: Supported kubernetes version is installed
+✅ ok: Supported helm version is installed
+✅ ok: Fluvio system charts are installed
+✅ ok: Can create service
+✅ ok: Can create customresourcedefinitions
+✅ ok: Can create secret
+✅ ok: Load balancer is up
+All checks passed!
+You may proceed with cluster startup
+next: run `fluvio cluster start`
+```
+
+## `fluvio cluster start`
+
+This command is used to start your own Fluvio cluster, with all the
+machinery needed to receive, process, and serve streaming messages.
+
+There are two main variations of this command. The default variation
+is invoked simply by `fluvio cluster start`. This will install Fluvio
+to a Kubernetes cluster, typically Minikube. The other variation is
+`fluvio cluster start --local`, which will start the cluster components
+as plain processes on your local machine.
+
+```
+fluvio-cluster-start
+Start a Fluvio cluster, locally or on Minikube
+
+USAGE:
+    fluvio cluster start [FLAGS] [OPTIONS]
+
+FLAGS:
+        --develop                  use local image
+        --skip-profile-creation
+        --sys                      installing sys
+        --local                    install local spu/sc(custom)
+        --tls                      Whether to use TLS
+        --skip-checks
+            Whether to skip pre-install checks, defaults to false
+
+        --setup
+            Tries to setup necessary environment for cluster startup
+
+    -h, --help                     Prints help information
+
+OPTIONS:
+        --chart-version <chart-version>
+            k8: use specific chart version [default: 0.6.0-beta.1]
+
+        --image-version <image-version>
+            k8: use specific image version
+
+        --registry <registry>
+            k8: use custom docker registry
+
+        --namespace <namespace>
+            k8 [default: default]
+
+        --group-name <group-name>
+            k8 [default: main]
+
+        --install-name <install-name>
+            helm chart installation name [default: fluvio]
+
+        --chart-location <chart-location>
+            Local path to a helm chart to install
+
+        --cloud <cloud>
+            k8 [default: minikube]
+
+        --spu <spu>
+            number of SPU [default: 1]
+
+        --rust-log <rust-log>
+            RUST_LOG options
+
+        --log-dir <log-dir>
+            log dir [default: /usr/local/var/log/fluvio]
+            
+        --domain <domain>                                        TLS: domain
+        --ca-cert <ca-cert>                                      TLS: ca cert
+        --client-cert <client-cert>
+            TLS: client cert
+
+        --client-key <client-key>                                TLS: client key
+        --server-cert <server-cert>
+            TLS: path to server certificate
+
+        --server-key <server-key>
+            TLS: path to server private key
+
+        --authorization-config-map <authorization-config-map>
+```
+
+Example usage:
```

Comment:
Use ### to get it in `toc` will help users to quickly jump to that section.

Reply:
Hmm, so when trying this, the TOC seems to get cluttered by the examples

![image](https://user-images.githubusercontent.com/4210949/118506101-06ad4f80-b6fb-11eb-9c5c-60e6f8b19432.png)

Reply:
@ajhunyady let me know if you prefer with or without examples in the TOC. I am leaning towards not putting them in TOC because of how cluttered it looks.

Reply:
One way to go around it is to downgrade the head to h5. `#####`

Reply:
Hmm I don't think I like the way that h5 looks in the new styling

![image](https://user-images.githubusercontent.com/4210949/118531062-657ec300-b713-11eb-8a7e-4e6bacf7b903.png)

It seems to imply bold+italics rather than to be a header. Plus I don't believe that it adds an anchor

Reply:
We can always change the styling, but I suggest we tackle these issues one at a time.

Reply:
BTW, sidebar only show h2/h3, so you can style with h4 `####`

Reply:
h4 also has an anchor.